src/Makefile.am

```diff
@@ -1,7 +1,8 @@
 ## Process this file with autoconf to produce configure.
 
-AM_CFLAGS   = -Wall -Wwrite-strings $(PTHREAD_CFLAGS)
-AM_CXXFLAGS = -Wall -Wwrite-strings -Woverloaded-virtual $(PTHREAD_CFLAGS)
+AM_CFLAGS   = $(HTM) -fgnu-tm -Wall -Wwrite-strings $(PTHREAD_CFLAGS)
+AM_CXXFLAGS = $(HTM) -fgnu-tm -Wall -Wwrite-strings -Woverloaded-virtual $(PTHREAD_CFLAGS)
+LIBS = -litm
```

Comment:
Using ```+=``` would work.
But ideally, you make this change in -litm in the configure.ac.  That's the only place that knows if libitm is really available in the system.